### PR TITLE
Update BETA blog post issue template for clarity and new assignee

### PR DESCRIPTION
**Pull request by lauracowen**

### CODE REVIEW

**Comment by lauracowen** [comment]
#run-libby-bot

File: .github/ISSUE_TEMPLATE/blog_post_beta.md [hunk 1]
```diff
@@ -3,19 +3,19 @@ name: Open Liberty BETA blog post
...
 
 Please provide the following information the week before the GA/beta date (to allow for review and publishing):
 
-1. Which Liberty feature does your update relate to?
+1. Which Liberty feature(s) does your update relate to?
     
    Human-readable name (eg WebSockets feature):
    
-   Feature name (eg websockets-1.0): 
+   Feature config name (eg websockets-1.0):
```
Comment (tevans78) [inline .github/ISSUE_TEMPLATE/blog_post_beta.md:18]: `Short Liberty feature names`?

**Review by tevans78 — APPROVED** [review]

**Comment by lauracowen** [comment]
#run-libby-bot

File: .github/ISSUE_TEMPLATE/blog_post_beta.md (hunk 1 above)
Comment (lauracowen) [reply .github/ISSUE_TEMPLATE/blog_post_beta.md]: Done

**Pull request merged**